packages/davinci-client/src/lib/config.slice.ts

```diff
@@ -37,8 +37,7 @@ export const configSlice = createSlice({
      */
     set(state, action: PayloadAction<DaVinciConfig>) {
       state.clientId = action.payload.clientId || '';
-      state.redirectUri =
-        action.payload.redirectUri || `${location.origin}/handle-redirect`;
+      state.redirectUri = action.payload.redirectUri || `${location.origin}/handle-redirect`;
```

Comment:
This second half the of the `||`, is `handle-redirect` expected to be there on the client? Or should this not be apart of the published code?

Reply:
Yes, the fallback is just a recommended default. The customer will need a dedicated route/page for this, or they provide a their own redirect URL.